autoscale/src/main/java/com/sequenceiq/periscope/service/ScalingActivityService.java

```diff
@@ -57,6 +57,9 @@ public class ScalingActivityService implements AuthorizationResourceCrnProvider,
     @Inject
     private ClusterService clusterService;
 
+    @Inject
```

Comment:
Can we remove this inject as it is not being used here?

Reply:
👍🏽